docs/kbs.yaml

```
@@ -0,0 +1,267 @@
+openapi: 3.0.1
+info:
+  title: Key Broker Service (KBS) API
+  description: HTTP based API for describing the communication between a a
+               Key Broker Client in a confidential guest and a trusted
```

For parity with KBS:

```suggestion
               Key Broker Client (KBC) in a confidential guest and a trusted
```

---

Fixed as well, thanks.